### PR TITLE
[Fixed #140] Added instructions for run locally

### LINKED ISSUE #140: Add Instructions to Run GCI'18 Website locally
**Actual Behaviour**

Currently, no instructions are present to run https://gci18.fossasia.org/ locally on your computer.

**Expected Behaviour**

Add Instructions for running https://gci18.fossasia.org/ locally on your computer.

**Helpful Links**

https://jekyllrb.com/

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by STEdirisuriya**
- [x] Included a screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
- Fixed https://github.com/fossasia/gci18.fossasia.org/issues/140
- I added instructions to Run GCI'18 Website locally 

image

If you need to see from my repo. Here's the link - https://github.com/STEdirisuriya/gci18.fossasia.org/blob/master/README.md

@fragm3 I have resolved your issue.
Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
 - Help to resolve merge conflicts (especially if there are several PRs at the same time, merge conflicts are common. Help the reviewers and solve merge conflicts to speed up the process.).
 - Merging Pull Requests should only happen if at least two contributors reviewed the PR and approved it.
 
+## How to run locally?
+- It is easy to run localy. Don't be affaraid!
```
Comment (akshatnitd) [inline README.md]: Correct the spelling of `afraid`

Comment (akshatnitd) [reply README.md]: Correct the spelling of `locally`

**Comment by STEdirisuriya** [comment]
@akshatnitd @fragm3 I did you requested. Please review now. Thank you!

File: README.md [hunk 2]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
...
 
+## How to run locally?
+- It is easy to run locally. Don't be afraid! :D
+- If you have git already installed, skip the step 3. If you have cloned your fork already skip steps 4 and 5. 
+1. First of all you need to complete some requirements. Install them if you don't have things in [this documentation](https://jekyllrb.com/docs/installation/).
+2. Open your terminal and install jekyll `gem install jekyll bundler`.
+3. Download and install git on your pc.
+4. Then go to [gci18.fossasia.org site](https://github.com/fossasia/gci18.fossasia.org) make a fork if you haven't already.
+5. After forking, open git bash and clone your repository into your pc. `git clone https://github.com/<_your-name-here_>/gci18.fossasia.org`.
+6. Go inside to your cloned repo, and open git bash. (If you have done 5th step do this `cd gci18.fossasia.org`).
```
Comment (Sid911) [inline README.md]: there is a ")' . Which I think is a typo

Comment (STEdirisuriya) [reply README.md]: No, here's it's not a typo. You could see there is a if statement. Thank you! 🙂

**Comment by STEdirisuriya** [comment]
@samagragupta I didn't see anywhere I add github rather than in a link. Could you please show where them are?

**Comment by STEdirisuriya** [comment]
@akshatnitd @fragm3 @saarthakchats @samagragupta @Sid911 Please could you have a look now. I changes all requested things. Thank you!

**Comment by samagragupta** [comment]
@STEdirisuriya Please make the changes at line 44 and 55.

**Comment by STEdirisuriya** [comment]
@samagragupta It's not from this PR. an earlier PR. should I change it?

File: README.md [hunk 3]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
 - Help to resolve merge conflicts (especially if there are several PRs at the same time, merge conflicts are common. Help the reviewers and solve merge conflicts to speed up the process.).
 - Merging Pull Requests should only happen if at least two contributors reviewed the PR and approved it.
 
+## How to run locally?
+- It is easy to run locally. Don't be afraid! :D
+- If you have Git already installed, skip the step 3. If you have cloned your fork already skip steps 4 and 5.
```
Comment (AakashMallik) [inline README.md]: skip **the** step --> skip step

Comment (AakashMallik) [inline README.md]: skip **the** step --> skip step

File: README.md [hunk 4]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
 - Help to resolve merge conflicts (especially if there are several PRs at the same time, merge conflicts are common. Help the reviewers and solve merge conflicts to speed up the process.).
 - Merging Pull Requests should only happen if at least two contributors reviewed the PR and approved it.
 
+## How to run locally?
+- It is easy to run locally. Don't be afraid! :D
+- If you have Git already installed, skip the step 3. If you have cloned your fork already skip steps 4 and 5. 
+1. First of all you need to complete some requirements. Install them if you don't have things in [this documentation](https://jekyllrb.com/docs/installation/).
+2. Open your terminal and install jekyll `gem install jekyll bundler`.
+3. Download and install git on your pc.
```
Comment (AakashMallik) [inline README.md]: git -> Git

File: README.md [hunk 5]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
 - Help to resolve merge conflicts (especially if there are several PRs at the same time, merge conflicts are common. Help the reviewers and solve merge conflicts to speed up the process.).
 - Merging Pull Requests should only happen if at least two contributors reviewed the PR and approved it.
 
+## How to run locally?
+- It is easy to run locally. Don't be afraid! :D
+- If you have Git already installed, skip the step 3. If you have cloned your fork already skip steps 4 and 5. 
+1. First of all you need to complete some requirements. Install them if you don't have things in [this documentation](https://jekyllrb.com/docs/installation/).
```
Comment (AakashMallik) [inline README.md]: have things in -> have **the programs listed** in

File: README.md [hunk 6]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
...
+2. Open your terminal and install jekyll `gem install jekyll bundler`.
+3. Download and install git on your pc.
+4. Go to [gci18.fossasia.org site](https://github.com/fossasia/gci18.fossasia.org) make a fork if you haven't already.
```
Comment (AakashMallik) [inline README.md]: make a fork --> fork it

File: README.md [hunk 7]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
...
+5. Open Git Bash and clone your repository into your pc. `git clone https://github.com/<_your-name-here_>/gci18.fossasia.org`.
+6. Go inside to your cloned repo, and open Git Bash.(If you have done 5th step do this `cd gci18.fossasia.org`).
```
Comment (AakashMallik) [inline README.md]: 5th step do this --> 5th step, do this

File: README.md [hunk 8]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
...
+7. Type `bundle install` to install dependencies.
+8. Type `bundle exec jekyll serve`.
```
Comment (AakashMallik) [inline README.md]: To do what?

File: README.md [hunk 9]
```diff
@@ -20,6 +20,21 @@ This is the repository for Fossasia's website for Google Code-In 2018, we at Fos
...
+2. Open your terminal and install jekyll `gem install jekyll bundler`.
+3. Download and install git on your pc.
+4. Go to [gci18.fossasia.org site](https://github.com/fossasia/gci18.fossasia.org) make a fork if you haven't already.
+5. Open Git Bash and clone your repository into your pc. `git clone https://github.com/<_your-name-here_>/gci18.fossasia.org`.
+6. Go inside to your cloned repo, and open Git Bash.(If you have done 5th step do this `cd gci18.fossasia.org`).
+7. Type `bundle install` to install dependencies.
+8. Type `bundle exec jekyll serve`. 
+9. You will see a server address. To view gci18.fossasia.org site, go to `localhost:4000` from your favourite browser.
+
+- Now the work is done.
```
Comment (AakashMallik) [inline README.md]: Sounds off. Change it something more friendly, like "You are all set" or "There you go, you have hosted GCI18 locally on your machine".

**Pull request merged**